hueniversitypy/theme_mcgill.py

```diff
@@ -30,43 +29,21 @@ def theme_mcgill():
             "title": {
                 "fontSize": 18,
                 "font": font,
-                "anchor": "start", # equivalent of left-aligned.
+                "anchor": "start", 
                 "fontColor": "#000000"
             },
+            # Axes fonts and sizes
             "axisX": {
-                "domain": True,
-                "domainColor": axisColor,
-                "domainWidth": 1,
-                "grid": False,
                 "labelFont": labelFont,
                 "labelFontSize": 12,
-                "labelAngle": 0, 
-                "tickColor": axisColor,
-                "tickSize": 5, # default, including it just to show you can change it
-                "titleFont": font,
-                "titleFontSize": 12,
```

Comment:
You need to keep the title font size! It is an important part of our theme and it is part of the tests